### PR TITLE
Cleanup the ccip e2e test

### PR DESCRIPTION
**Pull request by dimkouv**
While reading the test some things were not obvious.
This PR has some small improvements.

### CODE REVIEW

File: core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go [hunk 1]
```diff
@@ -59,91 +72,89 @@ func TestIntegration_OCR3Nodes(t *testing.T) {
...
-	// sourceChain map[uint64],  destChain [32]byte
-	var messageIDs = make(map[uint64]map[uint64][32]byte)
-	// map[uint64] chainID, blocks
-	var replayBlocks = make(map[uint64]uint64)
+	t.Logf("Initializing PingPong contracts")
 	pingPongs := initializePingPongContracts(t, universes)
 
+	// NOTE: messageIDs are populated in the sendPingPong function
+	var messageIDs = make(map[uint64]map[uint64][32]byte) // sourceChain->destChain->messageID
+	var replayBlocks = make(map[uint64]uint64)            // chainID -> blocksToReplay
```
Comment (makramkd) [inline core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go:143]: Not sure I like this paradigm of output parameters, seems error-prone.

File: core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go [hunk 2]
```diff
@@ -59,91 +72,89 @@ func TestIntegration_OCR3Nodes(t *testing.T) {
...
 
-	// replay the log poller on all the chains so that the logs are in the db.
+	// Wait for the oracles to come up.
+	// TODO: We need some data driven way to do this e.g. wait until LP filters to be registered.
+	time.Sleep(oraclesBootWaitTime)
+
+	// Replay the log poller on all the chains so that the logs are in the db.
 	// otherwise the plugins won't pick them up.
 	// TODO: this is happening too early, we need to wait for the chain readers to get their config
 	// and register the LP filters before this has any effect.
```
Comment (makramkd) [inline core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go]: Can remove this TODO

**Review by makramkd — APPROVED** [review]

File: core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go (hunk 1 above)
Comment (dimkouv) [reply core/services/ocr3/plugins/ccip_integration_tests/ocr3_node_test.go:143]: Same here, we'll fix that in a follow up

File: core/services/ccipcapability/oraclecreator/inprocess.go [hunk 3]
```diff
@@ -19,10 +19,11 @@ import (
...
 	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
 
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
 	commitocr3 "github.com/smartcontractkit/chainlink-ccip/commit"
 	execocr3 "github.com/smartcontractkit/chainlink-ccip/execute"
 	ccipreaderpkg "github.com/smartcontractkit/chainlink-ccip/pkg/reader"
-	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/types"
```
Comment (winder) [inline core/services/ccipcapability/oraclecreator/inprocess.go:28]: I guess this one should be grouped with ccipocr3

**Review by winder — APPROVED** [review]
Nice cleanup :+1:

**Pull request merged**